test failure is real
```
ERROR: corehq.apps.app_manager.tests.test_views:TestViews.test_advanced_module
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vendor/lib/python2.7/site-packages/mock.py", line 1224, in patched
    return func(*args, **keywargs)
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/test_views.py", line 181, in test_advanced_module
    'module_unique_id': module.unique_id,
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/test_views.py", line 123, in _test_status_codes
    response = self.client.get(reverse(name, kwargs=kwargs), follow=follow)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 529, in get
    **extra)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 333, in get
    return self.generic('GET', path, secure=secure, **r)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 409, in generic
    return self.request(**r)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 476, in request
    response = self.handler(environ)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 142, in __call__
    response = self.get_response(request)
  File "/vendor/lib/python2.7/site-packages/django/core/handlers/base.py", line 124, in get_response
    response = self._middleware_chain(request)
  File "/vendor/lib/python2.7/site-packages/django/core/handlers/exception.py", line 44, in inner
    response = response_for_exception(request, exc)
  File "/vendor/lib/python2.7/site-packages/django/core/handlers/exception.py", line 94, in response_for_exception
    response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())
  File "/vendor/lib/python2.7/site-packages/django/core/handlers/exception.py", line 143, in handle_uncaught_exception
    return callback(request, **param_dict)
  File "/mnt/commcare-hq-ro/corehq/apps/hqwebapp/views.py", line 122, in server_error
    request=request,
  File "/vendor/lib/python2.7/site-packages/django/template/backends/django.py", line 66, in render
    return self.template.render(context)
  File "/vendor/lib/python2.7/site-packages/django/template/base.py", line 206, in render
    with context.bind_template(self):
  File "/usr/local/lib/python2.7/contextlib.py", line 17, in __enter__
    return self.gen.next()
  File "/vendor/lib/python2.7/site-packages/django/template/context.py", line 232, in bind_template
    processors = (template.engine.template_context_processors +
  File "/vendor/lib/python2.7/site-packages/django/utils/functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/vendor/lib/python2.7/site-packages/django/template/engine.py", line 91, in template_context_processors
    return tuple(import_string(path) for path in context_processors)
  File "/vendor/lib/python2.7/site-packages/django/template/engine.py", line 91, in <genexpr>
    return tuple(import_string(path) for path in context_processors)
  File "/vendor/lib/python2.7/site-packages/django/utils/module_loading.py", line 27, in import_string
    six.reraise(ImportError, ImportError(msg), sys.exc_info()[2])
  File "/vendor/lib/python2.7/site-packages/django/utils/module_loading.py", line 23, in import_string
    return getattr(module, class_name)
ImportError: Module "corehq.util.context_processors" does not define a "skip_google_tag_manager" attribute/class
```

--- a/settings.py
+++ b/settings.py
@@ -1008,7 +1008,6 @@ TEMPLATES = [
                 'corehq.util.context_processors.current_url_name',
                 'corehq.util.context_processors.domain',
                 'corehq.util.context_processors.enterprise_mode',
-                'corehq.util.context_processors.skip_google_tag_manager',
                 'corehq.util.context_processors.js_api_keys',
                 'corehq.util.context_processors.websockets_override',
                 'corehq.util.context_processors.commcare_hq_names',

